rfc/rfc003-oauth2-authorization.md

```diff
@@ -251,7 +251,7 @@ The **sub** field in the JWT MUST be a known organization. It MUST have been reg
 
 **5.2.1.9 Purpose of use**
 
-The `purposeOfUse` field is used to define the scope of the access token. It contains a list of policy names. The policy names are defined by Bolts. All `purposeOfUse` entries from the accompanying authorization credentials MUST be listed. The resource server MUST be able to resolve the policy names from the access token.
+The `purposeOfUse` field MUST contain the scope of the access token. It contains a Bolt policy/service name. The `purposeOfUse` entries from the accompanying authorization credentials MUST match. The resource server MUST be able to resolve the policy name from the access token.
```

Comment:
>The `purposeOfUse` entries
This suggests an authorization credential can contain multiple entries for purposeOfUse but it is a single value?